### ISSUE TITLE
Update to version 3.0 of media-insights-engine

### ISSUE THREAD

**Issue by aburkleaux-amazon** [aws-samples/aws-media-insights-content-localization#117]

**Comment by aburkleaux-amazon** [comment]
Create vocabulary API doesn't work in MIE 3.0: regression due to change in workflowapi policy:

https://github.com/awslabs/aws-media-insights-engine/pull/430

This PR removes permission for the workflow API to read from the dataplane bucket

**Comment by aburkleaux-amazon** [comment]
Update to MIE 3.0 is not possible, will need to wait for next version with fix.

**Comment by aburkleaux-amazon** [comment]
I believe this is included in the PR (#139 ) to merge from CAS

**Comment by ianwow** [comment]
Yes, it's in that PR. Here's the line number: https://github.com/aws-samples/aws-media-insights-content-localization/blob/dc945e895c0f4d4a6159e383368c9ce9ce909dc8/deployment/aws-content-localization.yaml#L27

This issue will automatically close when that PR is merged.
